.github/workflows/docker-build-tag.yml

```diff
@@ -97,27 +97,29 @@ jobs:
 
           export GH_PUSH_TOKEN
           export RELEASE_VERSION
+          export SERVICE
 
-          # Check if release branch exists
+          # Check if release branch exists in helm-charts repository
           REPO_OWNER=kubeshop
           REPO_NAME=helm-charts
-          RELEASE_BRANCH_NAME=release-$RELEASE_VERSION
+          RELEASE_BRANCH_NAME=release/dashboard/$RELEASE_VERSION
           echo $RELEASE_VERSION
           STATUS=$(curl -s -o /dev/null -w '%{http_code}' -H "${AUTH_HEADER}" "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/branches/${RELEASE_BRANCH_NAME}")
 
           if [ "${STATUS}" -eq 404 ]; then
             git fetch origin main
             git checkout main
             cd ./scripts
-            ./chart_releaser.sh --helm-chart-folder testkube-dashboard
+            ./chart_releaser.sh --helm-chart-folder testkube-$SERVICE
```

Comment:
Is the `testkube-` common prefix, so it can use `testkube-$SERVICE`, or is `testkube-dashboard` atomic, so it shouldn't use var to detect part of it?

Reply:
yes, it's common. I will replace it for the api and operator as well.